gnomad_qc/v4/sample_qc/sex_inference.py

```diff
@@ -347,8 +345,24 @@ def compute_sex(
     :param min_af: Minimum alternate allele frequency to be used in f-stat calculations. Default is 0.001.
     :param f_stat_cutoff: f-stat to roughly divide 'XX' from 'XY' samples. Assumes XX samples are below cutoff and XY
         are above cutoff. Default is -1.0.
-    :return: Table with inferred sex annotation.
+    :return: Table with imputed ploidies.
     """
+    if (high_cov_per_platform or high_cov_all_platforms) and platform_ht is None:
+        raise ValueError(
+            "'platform_ht' must be defined if 'high_cov_per_platform' or 'high_cov_all_platforms' is True!"
+        )
+
+    if high_cov_per_platform and high_cov_all_platforms:
+        raise ValueError(
+            "Only one of 'high_cov_per_platform' or 'high_cov_all_platforms' can be True!"
+        )
+
+    if high_cov_intervals and high_cov_all_platforms:
+        logger.warning(
+            "Both 'high_cov_intervals' and 'high_cov_all_platforms' are True, high coverage intervals will be defined"
+            " as intervals that match high coverage criteria within all platforms."
+        )
```

Comment:
is this warning no longer relevant?

Reply:
It's just saying that if high_cov_intervals and high_cov_all_platforms are both set it will default to performing `high_cov_all_platforms`. It's not really relevant because of the `add_mutually_exclusive_group` below, but I think it's OK to keep